client/rest/test/plugins/rosetta/accountRoutes_spec.js

```diff
@@ -121,6 +144,80 @@ describe('account routes', () => {
 	};
 
 	const addAccountSuccessTests = (route, options) => {
+		const createTransactionJson = (mosaics, signerPublicKey, recipientAddress, isEmbedded = false) => {
+			const facade = new SymbolFacade('testnet');
+			const transaction = facade.transactionFactory.create({
+				type: 'transfer_transaction_v1',
+				recipientAddress,
+				signerPublicKey,
+				mosaics,
+				fee: 100
+			});
+
+			const hashPropertyName = isEmbedded ? 'aggregateHash' : 'hash';
+			return {
+				transaction: convertTransactionSdkJsonToRestJson(transaction.toJson()),
+				meta: {
+					[hashPropertyName]: TRANSACTION_HASH,
+					height: '0',
+					index: 1
+				}
+			};
+		};
+
+		const createAggregateTransactionJson = signerPublicKey => {
+			const facade = new SymbolFacade('testnet');
+			const aggregateTransaction = facade.transactionFactory.create({
+				type: 'aggregate_complete_transaction_v2',
+				signerPublicKey,
+				fee: 25n,
+				deadline: 12345n
+			});
+
+			return {
+				transaction: convertTransactionSdkJsonToRestJson(aggregateTransaction.toJson()),
+				meta: {
+					hash: TRANSACTION_HASH,
+					height: '0',
+					index: 1
+				}
+			};
+		};
+
+		const assertIncludeMempoolAccountTest = async (transactions, expectedCoins, includeMempool = true) => {
```

Comment:
i think there was miscommunication again?

these should be under `describe('/account/coins'` since they're only testing that route (not both). currently, these are being added twice - once under `describe('/account/coins` and again under `describe('/account/balance`. 

only the other (two?) routes that are generic and applicable to both routes should be added in addAccountSuccessTests. even if they have the same `it` names, they'll be testing the appropriate (different) routes.

Reply:
Updated to what you said.  After I debug this realized that the include_mempool gets striped out when we create the request object(`` Request.constructFromObject(req.body);``).  So its not an issue and I don't need to add the tests for ``/account/balance``.